app/org/sagebionetworks/bridge/services/StudyService.java

```diff
@@ -221,10 +302,12 @@ public Study createSynapseProjectTeam(String synapseUserId, Study study) throws
         }
 
         // then check if the user id exists
-        try {
-            synapseClient.getUserProfile(synapseUserId);
-        } catch (SynapseNotFoundException e) {
-            throw new BadRequestException("Synapse user Id is invalid.");
+        for (String userId : synapseUserIds) {
+            try {
+                synapseClient.getUserProfile(userId);
+            } catch (SynapseNotFoundException e) {
+                throw new BadRequestException("Synapse user Id is invalid.");
```

Comment:
So, we could add the ID to these exceptions so the user knows which one has the error.